dissect/target/loaders/mqtt.py

```diff
@@ -62,12 +67,79 @@ def _read(self, offset: int, length: int, optimization_strategy: int = 0) -> byt
         return data
 
 
+class MQTTDiagnosticLine:
+    def __init__(self, connection: MQTTConnection, total_peers: int):
+        self.connection = connection
+        self.total_peers = total_peers
+        self._columns, self._rows = os.get_terminal_size(0)
+        atexit.register(self._detach)
+        self._attach()
+
+    def _attach(self) -> None:
+        sys.stderr.write(f"\0337\033[r\0338\033D\033M\0337\033[1;{self._rows - 1}r\0338")
+
+    def _detach(self) -> None:
+        sys.stderr.write(f"\0337\033[{self._rows};1H\033[K\033[r\0338")
+        sys.stderr.flush()
+
+    def display(self) -> None:
+        prefix = "\x1b[44m\x1b[37m\r"
+        suffix = "\x1b[0m"
+        separator = "\x1b[41m\x1b[1m"
+        logo = "TARGETD"
+        start = time.time()
+        t2 = start
+        mark = start
+        _bytes = 0
+        subtract = 0
```

Comment:
```suggestion
        prefix = "\x1b[44m\x1b[37m\r"
        suffix = "\x1b[0m"
        separator = "\x1b[41m\x1b[1m"
        logo = "TARGETD"

        start = time.time()
        t2 = start
        mark = start

        _bytes = 0
        subtract = 0

```

Some newlines to improve readability